src/test_least_square.cc

```diff
@@ -59,7 +59,7 @@ int main(int argc, char *argv[]) {
   /*****************************
      2. load data from file
   *****************************/
-  std::cout << "% Start to load data!" << std::endl;
+  std::cout << "% Start to load data! This may take a few minutes if you use our test." << std::endl;
```

Comment:
No "!" Do not use unless something is important.

"Our default test files are large, so it will take a while to load."